app/views/articles/send_last_users_articles.js.erb

```diff
@@ -0,0 +1,2 @@
+$('#task-form').html("<%= j (render 'processing') %>");
```

Comment:
Using js.erb is not a good idea. Mixing html with js is not good. And mixing rails with html, with js.. you can imagine haha

The right way to do this is using $.ajax and return an html in the controller